src/microbe_stage/organelle_components/SlimeJetComponent.cs

```diff
@@ -41,6 +41,8 @@ public void OnAttachToCell(PlacedOrganelle organelle)
         parentOrganelle = organelle;
 
         organellePosition = Hex.AxialToCartesian(organelle.Position);
+
+        queuedForce = Vector3.Zero;
```

Comment:
Did you mean to open this as a draft? This is the wrong variable to modify (per the discussion on the issue). You need to modify `MicrobeControl.QueuedSlimeSecretionTime` in `MicrobeColony.OnColonyMemberAdded`.

Reply:
Oops